test/Scripts.Integration.Test/integration-test.ps1

```diff
@@ -82,18 +82,16 @@ If (-not(Test-Path -Path "$(GetNewProjectPath)"))
     ./test/Scripts.Integration.Test/configure-sentry.ps1 "$UnityPath" -Platform $Platform -CheckSymbols
 }
 
+# Support rebuilding the integration test project. I.e. if you make changes to the SmokeTester.cs during
 If ($Rebuild -or -not(Test-Path -Path $(GetNewProjectBuildPath)))
 {
     Write-Host "Building Project"
 
-    If (("iOS", "Android-Export") -contains $Platform)
+    If ("iOS" -eq $Platform)
     {
-        # Workaround for having `exportAsGoogleAndroidProject` remain `false` in Unity 6 on first build
+        # We're exporting an Xcode project and building that in a separate step.
         ./test/Scripts.Integration.Test/build-project.ps1 -UnityPath "$UnityPath" -UnityVersion $UnityVersion -Platform $Platform
-        Remove-Item -Path $(GetNewProjectBuildPath) -Recurse -Force -Confirm:$false
-
-        ./test/Scripts.Integration.Test/build-project.ps1 -UnityPath "$UnityPath" -UnityVersion $UnityVersion -Platform $Platform
-        & "./scripts/smoke-test-$($Platform -eq 'iOS' ? 'ios' : 'android').ps1" Build -IsIntegrationTest -UnityVersion $UnityVersion
```

Comment:
we don't need the Android stuff here anymore?

Reply:
That script gluing all the scripts together and letting us "run CI locally".
And no, we're not exporting the Gradle project to manually build it on a different runner. We're relying on the Android Dev environment Unity comes with.